cmd/sandbox/start.go

```diff
@@ -137,35 +131,35 @@ func startSandbox(ctx context.Context, cli docker.Docker, reader io.Reader) (*bu
 		volumes = append(volumes, *vol)
 	}
 
-	if len(sandboxConfig.DefaultConfig.Version) > 0 {
-		isGreater, err := util.IsVersionGreaterThan(sandboxConfig.DefaultConfig.Version, flyteMinimumVersionSupported)
+	if sandboxConfig.DefaultConfig.Version != "latest" {
+		isGreater, err := util.IsVersionGreaterThan(sandboxConfig.DefaultConfig.Version, "v0.10.0")
 		if err != nil {
 			return nil, err
 		}
 		if !isGreater {
-			logger.Infof(ctx, "version flag only supported after with flyte %s+ release", flyteMinimumVersionSupported)
-			return nil, fmt.Errorf("version flag only supported after with flyte %s+ release", flyteMinimumVersionSupported)
-		}
-		if err := githubutil.GetFlyteManifest(sandboxConfig.DefaultConfig.Version, flyteManifest); err != nil {
-			return nil, err
+			return nil, fmt.Errorf("version flag only supported with flyte v0.10.0+ release")
 		}
-
-		if vol, err := mountVolume(flyteManifest, generatedManifest); err != nil {
+		sha, err := githubutil.GetSHAFromVersion(sandboxConfig.DefaultConfig.Version, "flyte")
+		if err != nil {
 			return nil, err
-		} else if vol != nil {
-			volumes = append(volumes, *vol)
 		}
-
+		sandboxConfig.DefaultConfig.Version = fmt.Sprintf("%s-%s", dind, sha)
+	} else {
+		sandboxConfig.DefaultConfig.Version = dind
 	}
```

Comment:
Why overwrite this config. Also the ```sandboxConfig.DefaultConfig.Version != "latest"``` L134  else part is ignoring and overwriting . This seems wierd logic